<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>regprof</class>
 <widget class="QDialog" name="regprof">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>register</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/img_file/register.png</normaloff>:/img_file/register.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="regprof_2">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>register</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lab">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>+statut</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_2prenom"/>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_4cin"/>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEdit_1nom"/>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Inscription</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
